frontend/tailwind.config.js

```diff
@@ -1,6 +1,8 @@
 module.exports = {
-   purge: ['./src/**/*.{js,jsx,ts,tsx}', './public/index.html'],
-    darkMode: false, // or 'media' or 'class'
+    content: [
```

Comment:
Why the tailwind changes?

Reply:
unrelated to fix but resolves some build errors as tailwind no longer uses purge as a library.